### DISCUSSION TITLE
Multiple warnings since Mastercontainer upgrade, mostly webserver can't connect to itself.

### DISCUSSION THREAD

**Discussion by msrv** [nextcloud/all-in-one#4998] [Questions] [answered]
### Steps to reproduce
1. Upgrade mastercontainer to v9.1.0
2. Go to NC 29 Admin interface
3.

### Expected behavior 
No warnings shown as there were no warnings before the upgrade

### Actual behavior 
Multiple warnings given:
`Could not check that the data directory is protected. Please check manually that your server does not allow access to the data directory. To allow this check to run you have to make sure that your webserver can connect to itself. Therefor it must be able to resolve and connect to at least one its `trusted_domains` or the `overwrite.cli.url`.
Your webserver is not set up to serve `.js.map` files. Without these files, JavaScript Source Maps won't function properly, making it more challenging to troubleshoot and debug any issues that may arise.
Could not check for JavaScript support via any of your `trusted_domains` nor `overwrite.cli.url`. This may be the result of a server-side DNS mismatch or outbound firewall rule. Please check manually if your webserver serves `.mjs` files using the JavaScript MIME type. To allow this check to run you have to make sure that your webserver can connect to itself. Therefor it must be able to resolve and connect to at least one its `trusted_domains` or the `overwrite.cli.url`.
Could not check if your web server properly resolves the OCM and OCS provider URLs. To allow this check to run you have to make sure that your webserver can connect to itself. Therefor it must be able to resolve and connect to at least one its `trusted_domains` or the `overwrite.cli.url`.
One or more mimetype migrations are available. Occasionally new mimetypes are added to better handle certain file types. Migrating the mimetypes take a long time on larger instances so this is not done automatically during upgrades. Use the command `occ maintenance:repair --include-expensive` to perform the migrations.`

### Host OS 
Ubuntu jammy

#### Nextcloud AIO version 
9.1.0

#### Current channel 
latest

**Comment by szaimen** [comment, marked as answer]
Hi, most likely does your router not support NAT loopback. You can work around this by following https://github.com/nextcloud/all-in-one#how-can-i-access-nextcloud-locally

**Comment by evanraalte** [comment]
I had the same problem, and in my case the problem was fail2ban, that had blocked the IP of the server, effectively inhibiting the NAT loopback.
